app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from __future__ import annotations
+from typing import List
+
+
+class Animal:
+    alive: List[Animal] = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
```

Comment:
This is a good use of string interpolation for constructing the string representation of the object.